components/dialog/test/dialog.test.js

```diff
@@ -17,7 +17,8 @@ describe('d2l-dialog', () => {
 	describe('focus management', () => {
 
 		it('should focus on close button if no focusable elements inside', async() => {
-			const el = await fixture(html`<d2l-dialog opened>not focusable</d2l-dialog>`);
+			const el = await fixture(html`<d2l-dialog>not focusable</d2l-dialog>`);
+			el.opened = true;
```

Comment:
As discussed previously, these types of tests where an event is fired as part of boot-up need to be adjusted to control when the event gets fired themselves.